kairon/actions/definitions/form_validation.py

```
@@ -91,6 +93,20 @@ async def execute(self, dispatcher: CollectingDispatcher, tracker: Tracker, doma
+                bot_name = Bot.objects(id=self.bot).only("name").get().name
+
+                await MailUtility.format_and_send_mail(mail_type="action_failure",
+                                                       email=tracker.sender_id,
+                                                       stack_trace=str(e),
+                                                       slot_values=tracker.current_slot_values(),
+                                                       first_name="Kairon Team",
+                                                       bot_name=bot_name,
+                                                       action_name=self.name,
+                                                       user_query_history=tracker.latest_message.get('text')
+                                                       )
+            except Exception as mail_err:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)